lang/java/avro/pom.xml

```diff
@@ -198,6 +198,11 @@
       <artifactId>joda-time</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.github.luben</groupId>
+      <artifactId>zstd-jni</artifactId>
+      <optional>true</optional>
```

Comment:
I think that most of the compression dependencies should be <optional>.  Its extra baggage that is not useful in any case where we aren't writing out or reading files.  It would be more consistent with the other codecs to remove this.